canonical-web-and-design/charmhub.io
# Add side navigation basic functionality

## Done

- Add basic functionality to the side navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/screen/5e39844ac39722c6d40e682a) - note that I used the future vanilla side navigation
- Test it with javascript disabled as well - here is a [guide to disable it](https://developers.google.com/web/tools/chrome-devtools/javascript/disable)


## Issue / Card

Fixes #3
